recipes/brpc/all/CMakeLists.txt

```diff
@@ -0,0 +1,23 @@
+cmake_minimum_required(VERSION 2.8.11)
+project(cmake_wrapper)
+
+include(conanbuildinfo.cmake)
+conan_basic_setup()
+
+# disable picking up system-installed libraries with find_library
+set(CMAKE_FIND_USE_CMAKE_SYSTEM_PATH FALSE)
+
+# we need to set the executable manually since that seems to be missing in
+# the generated FindProtobuf.cmake in the protobuf conan package
+find_program(PROTOBUF_PROTOC_EXECUTABLE NAMES protoc)
+# the brpc build uses both variants of protobuf include directories
+set(PROTOBUF_INCLUDE_DIRS "${CONAN_PROTOBUF_ROOT}/include")
+set(PROTOBUF_INCLUDE_DIR "${CONAN_PROTOBUF_ROOT}/include")
```

Comment:
Won't change anything, but I think you can use `CONAN_PROTOBUF_INCLUDE_DIRS` (or something similar).